src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -2,6 +2,7 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_TASK_DATETIME_FOR_RECURRING_TASK;
+import static seedu.address.logic.commands.AddTaskCommand.MESSAGE_DUPLICATE_TASK;
```

Comment:
I think maybe we should move these to commons->core->messages?

Reply:
Moved!